scripts/apps/search/services/SearchService.js

```diff
@@ -596,6 +604,31 @@ export function SearchService($location, gettext, config, session) {
     };
 
     /**
+     * Returns the query criteria for a single item while keeping
```

Comment:
I'm not sure that this documentation style is correct for the methods. Best way to make sure it's ok is to run `grunt docs` and see if it displays.  Let me know if you've already did and it's working.

I would think you need to take the approach like all the other methods, such as the one below, for example (`doesSearchAgainstRepo`)